types/react-relay/test/react-relay-tests.tsx

```diff
@@ -714,3 +714,45 @@ requestSubscription(
 
 ReactRelayContext.Consumer.prototype;
 ReactRelayContext.Provider.prototype;
+
+const MyRelayContextProvider: React.FunctionComponent = children => {
+    return (
+        <ReactRelayContext.Provider
+            value={{
+                environment: modernEnvironment,
+            }}>
+            {children}
+        </ReactRelayContext.Provider>
+    );
+};
+
+const MyRelayContextConsumer: React.FunctionComponent = () => {
+    const relayContext = React.useContext(ReactRelayContext);
+    if (!relayContext || !relayContext.environment) {
+        return null;
+    }
+
+    return (
+        <QueryRenderer<MyQuery>
+            environment={relayContext.environment}
```

Comment:
So to be clear, this test failed perviously, yeah?

Reply:
yep! if the change in the type defs is reverted this will start throwing an error

Reply:
Ace, thanks for confirming 👍